test/io-hooks/test.k

```diff
@@ -0,0 +1,14 @@
+module TEST
+  imports STRING
+  imports INT
+  imports K-IO
+
+  syntax IOString ::= readFileAux(IOInt) | readFile(String) [function, impure]
+
+  rule readFileAux(#ENOENT) => "File not found"
+  rule readFileAux(FD:Int) =>
+    #let TXT = #read(FD, 10) #in
+    #let _ = #close(FD) #in TXT
+
+  rule readFile(F) => readFileAux(#open(F))
```

Comment:
Added a simple read test. Testing write would be a bit more tricky and don't have time to add one now. @ana-pantilie if you think we should add a write test too, let me know here and I can add it when I get back.

Reply:
@goodlyrottenapple maybe add it in a subsequent PR, after you get back. We'll discuss this then.